topics/snuba-uptime-results.yaml

```diff
@@ -0,0 +1,18 @@
+pipeline: uptime
+description: uptime check results for snuba
+services:
+  producers:
+    - getsentry/sentry
+  consumers:
+    - getsentry/snuba
+schemas:
+  - version: 1
+    compatibility_mode: backward
+    type: json
+    resource: snuba-uptime-results.v1.schema.json
```

Comment:
I think the schemas are identical, so can we just point to `uptime-results.yaml` here and avoid the duplication?

Reply:
they aren't identical, as the new schema contains organization_id, project_id, retention_days, and region_slug which the old one does not contain.

we could alternatively update the old schema to contain these values i suppose.

Reply:
👎 to having the other schema include these values. They won't be used since the uptime checker isn't aware of org / project_id (and can't be since the requests are de-duplicated right now)